packages/slate-react/src/constants/environment.js

```diff
@@ -92,8 +92,12 @@ export const IS_FIREFOX = BROWSER === 'firefox'
 export const IS_SAFARI = BROWSER === 'safari'
 export const IS_IE = BROWSER === 'ie'
 
+export const IS_ANDROID = OS === 'android'
 export const IS_IOS = OS === 'ios'
 export const IS_MAC = OS === 'macos'
 export const IS_WINDOWS = OS === 'windows'
 
+export const IS_MOBILE = IS_ANDROID || IS_IOS
+export const IS_DESKTOP = !IS_MOBILE
```

Comment:
I think since these two are kind of leaky (in the event of other types of mobile OS's) we should instead be specific and use `IS_ANDROID` and `IS_IOS` in the one place we're going to use this for now. Later if we want to make `IS_MOBILE/IS_DESKTOP` more fleshed out we can do that.

Reply:
Thanks, got it fixed.